packages/x-components/src/design-system/components/suggestion/default.scss

```diff
@@ -9,7 +9,6 @@
   align-items: var(--x-string-align-items-suggestion-default);
```

Comment:
1. I would add an example of this to the design system, like I asked in the suggestions group.
2. Same issue with large texts.

```html
    <article class="x-list x-list--wrap x-list--gap-06 x-list--align-start">
      <h2 class="x-title2">Default + Tag</h2>

      <div class="x-suggestion x-tag">
        <span class="x-suggestion__query">Suggestion</span>
      </div>

      <div class="x-suggestion x-tag x-suggestion--matching">
        <span class="x-suggestion__query">
          Suggestion
          <span class="x-suggestion__matching-part">Query</span>
        </span>
      </div>

      <div class="x-suggestion x-tag x-suggestion--matching">
        <SearchIcon />
        <span class="x-suggestion__query">
          Suggestion
          <span class="x-suggestion__matching-part">Query</span>
        </span>
      </div>

      <div class="x-suggestion x-tag x-suggestion--matching">
        <span class="x-suggestion__query">
          Suggestion
          <span class="x-suggestion__matching-part">Query</span>
        </span>
        <SearchIcon />
      </div>

      <div class="x-suggestion x-tag x-suggestion--matching">
        <SearchIcon />
        <span class="x-suggestion__query">
          Suggestion extra large name
          <span class="x-suggestion__matching-part">Query</span>
        </span>
      </div>

      <div class="x-suggestion x-tag x-suggestion--matching">
        <span class="x-suggestion__query">
          Suggestion extra large name
          <span class="x-suggestion__matching-part">Query</span>
        </span>
        <SearchIcon />
      </div>
    </article>
```
![image](https://user-images.githubusercontent.com/68222542/149500197-f1cc21fc-eb1b-4d38-90b8-fa1cac679619.png)

```

Reply:
About the previous question and this one, I talked with @tajespasarela yesterday about this and he said that it is enough to have the tag examples but i can add new examples

Reply:
I added the example, but I'm not sure about the second one, maybe we can add a utility to do ellipsis in the text?